### PR TITLE
Dev brady

### PR DESCRIPTION
**Pull request by BradyBangasser**

### CODE REVIEW

**Review by CalSch — APPROVED** [review]
amazing fantastic incredible

**Comment by CalSch** [comment]
someone please review aaaaa

File: pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    title: "Team History",
+    value: `We were founded in 2010 (yes, we're ${
+      new Date().getFullYear() - 2010
+    } years old) and we have competed in the FRC Robotics Competition every year since. We operate out of Mounds View High School and the majority of our students attend there as well.`,
+  },
+  {
+    title: "Our Mission",
+    value:
+      "Are main goal is to improve the skills of our team members (Mentors and Students) to help them be more successful in their current and future endeavers in STEM.",
```
Comment (ajess139) [inline pages/about.tsx:20]: our vs are

File: pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  },
+  {
+    title: "Key Team Values",
+    value: (
+      <ul style={{ textAlign: "left" }}>
+        <li>It{"'s"} not stupid if it works</li>
```
Comment (ajess139) [inline pages/about.tsx:26]: I think we have a different set of values somewhere, actually.  So perhaps comment this out until we have the right set.

Comment (BradyBangasser) [reply pages/about.tsx:26]: Fixed

File: pages/about.tsx (hunk 1 above)
Comment (BradyBangasser) [reply pages/about.tsx:20]: I just commented out the entire about page

Comment (BradyBangasser) [reply pages/about.tsx:20]: I did fix it too

**Review by ajess139 — APPROVED** [review]

**Pull request merged**